clvm/operators.py

```diff
@@ -35,6 +36,17 @@
 }
 
 
+def default_unknown_op(op: bytes, arguments: CLVMObject) -> Tuple[int, CLVMObject]:
```

Comment:
What's the motivation for this implementation? (Even / Odd)

Reply:
My understanding is that it will enable us, to some extent, extend the instruction set in the future, and still have old clients have a reasonable alternative behavior and cost.

If we want to have a new instruction that is a hard failure for clients that don't understand it, half the opcode space give you that. If we want a new instruction that is a no-op for clients that don't understand it, that's the other half of the op-code space.

If we want the no-op to have a certain cost (within some limits), that opcode is available too

Reply:
We might want some opcodes that have a cost that depends upon the inputs in some way that somewhat mirrors how complicated costs work in other opcodes.

Reply:
I also think it might make sense to move this logic to `chia-blockchain` (ie. allow the long tails of the dialects to be defined there).

Reply:
That does make testing with `brun` more challenging though. I wonder if `brun` should optionally take a `-s`-like function that defines a full path to a class that completely defines the dialect.

Reply:
Along these lines, there's an argument to be made that everything in `more_ops` could actually belong in `chia-blockchain` (although it gets trickier when rust is involved), and then we move the corresponding `brun` tests there. It seems to make more sense when `clvm` core is relatively stable.